src/webportal/src/app/vc/vc.component.ejs

```diff
@@ -43,15 +47,30 @@
                 <td>
                   <%= data[vcName]['numJobs'] %>
                 </td>
+                <td width="8%">
+                  <a onclick="changeVcState('<%= vcName %>', '<%= data[vcName]['status'] %>')" class="state-vc state-<%= data[vcName]['status'].toLowerCase() %> <%= admin ? '' : 'state-vc-ordinary' %>">
+                    <%= data[vcName]['status'].toLowerCase().replace(/( |^)[a-z]/g, (L) => L.toUpperCase()) %>
+                    <% if (admin && vcName !== 'default') {%>
+                      <span class="tips"></span>
+                    <% } %>
+                  </a>
+                </td>
                 <td>
                   <a href="view.html?vcName=<%= vcName %>">View Jobs</a>
                   &nbsp;&nbsp;|&nbsp;&nbsp;
                   <a href="<%= yarnWebPortalUri %>/cluster/scheduler?openQueues=<%= vcName %>" target="_blank">Go to Yarn Page</a>
+                  <% if (admin) { %>
+                    &nbsp;&nbsp;|&nbsp;&nbsp;
+                    <a href="javscript:;" onclick="deleteVcItem('<%= vcName %>')" class="<%= vcName=='default' ? 'default' : '' %>">Delete</a>
```

Comment:
Use link button instead, `href="javascript:;"` will cause warnings in browsers with strict CSP enabled.